### PR TITLE
docs(www): update graphql starter documentation to reflect graphql_co…

### PR DESCRIPTION
**Pull request by cruno91**
…mpose 2.0.0 and graphql 4.6.0

This pull request is for: (mark with an "x")

- [ ] `examples/*`
- [ ] `modules/next`
- [ ] `packages/next-drupal`
- [ ] `starters/basic-starter`
- [ ] `starters/graphql-starter`
- [x ] Other

GitHub Issue: https://github.com/chapter-three/next-drupal/issues/596

- [ x] I need help adding tests. (mark with an "x")

## Describe your changes

Updated the docs to reflect updated patches for drupal/subrequests & drupal/decoupled_router and the changes in how drupal/graphql_compose is configured in version 2.0.0.

### CODE REVIEW

File: www/content/tutorials/graphql/enable-modules.mdx [hunk 1]
```diff
@@ -8,13 +8,20 @@ group: Quick Start (GraphQL)
 1. Visit `/admin/modules`.
 2. Enable the following modules: **Next.js** and **Next.js GraphQL**.
 
+## Configure GraphQL Compose
+
+1. Visit `/admin/config/graphql_compose`.
+2. Enable "GraphQL", "Single Query", "Edge Query", & "Loading by Route" for the Article content type.
```
Comment (JohnAlbin) [inline www/content/tutorials/graphql/enable-modules.mdx:14]: "Edge Query" is not an option unless "GraphQL Compose: Edges" is enabled, which the instructions do not indicated we need to do.

"Loading by Route" is not an option unless "GraphQL Compose: Routes" is enabled, which the instructions do not indicated we need to do.

So… we need to update the step above "2. Enable the following modules" to include those 2 modules. Are there other modules that should also be enabled?

**Review by JohnAlbin — CHANGES_REQUESTED** [review]
This is looking really good! But I noticed the "Enable the following modules" step also needs updating.

File: www/content/tutorials/graphql/enable-modules.mdx (hunk 1 above)
Comment (cruno91) [reply www/content/tutorials/graphql/enable-modules.mdx:14]: You're right, I forgot the module was split up a bit between beta and 2.x releases.  I'll try to get a commit in for this this week if I can clear some time.

Comment (cruno91) [reply www/content/tutorials/graphql/enable-modules.mdx:14]: @JohnAlbin it took a while for me to get back to it, but I've updated the list of modules for the GrpahQL example docs.

**Review by JohnAlbin — APPROVED** [review]

**Pull request merged**